.travis.yml

```diff
@@ -14,7 +14,7 @@ sudo: false  # use the new container-based Travis infrastructure
 dist: trusty # Ubuntu 14.04
 
 mono:
-  - latest
+  - nightly
```

Comment:
possibly also add `- latest` to the matrix, and then allow failures on `nightly` because of how fragile that could be.

Can look at the [FSharp.Data](https://github.com/fsharp/FSharp.Data/blob/master/.travis.yml) travis config for how to do that.